src/Miew.js

```diff
@@ -2934,6 +2922,59 @@ Miew.prototype.setPivotAtom = function (atom) {
   this.dispatchEvent({ type: 'transform' });
 };
 
+Miew.prototype.setPivotSubset = (function () {
+  const _center = new THREE.Vector3(0.0, 0.0, 0.0);
+
+  function _includesInCurSelection(atom, selectionBit) {
+    return atom._mask & (1 << selectionBit);
+  }
+
+  function _includesInSelector(atom, selector) {
+    return selector.selector.includesAtom(atom);
+  }
+
+  return function (selector) {
+    const pos = this._gfx.pivot.position;
+    let vCount = 0;
+
+    let includesAtom;
+    if (selector) {
+      includesAtom = _includesInSelector;
```

Comment:
Also, I want to see here the ternary operator. It simplifies the code.

Reply:
Thank you for correction. Done.